client/src/pages/Login/index.tsx

```diff
@@ -1,5 +1,84 @@
+import { Button, TextField } from '@mui/material';
+import { useFormik } from 'formik';
+import { Link } from 'react-router-dom';
+import * as yup from 'yup';
+import './style.css';
+
+const validationSchema = yup.object({
+  email: yup
+    .string()
+    .email('Enter a valid email')
+    .required('Email is required'),
+  password: yup
+    .string()
+    .min(8, 'Password should be of minimum 8 characters length')
+    .required('Password is required'),
+});
+
 function Login() {
-  return (<h1>Login</h1>);
+  const formik = useFormik({
+    initialValues: {
+      email: '',
+      password: '',
+    },
+    validationSchema,
+    onSubmit: (values) => {
+      alert(JSON.stringify(values, null, 2));
+    },
+  });
+
+  return (
+    <div className="loginPage">
+      <div
+        className="loginImg"
+      >
+        <div className="decsLogin">
+          <h3>LOG IN</h3>
+
+          <h2>
+            The better way
+            to
+            buy or sell a used car
+          </h2>
+          <p>Great Value | Trusted Quality | All Online</p>
+        </div>
+      </div>
+      <div className="formPage">
+        <form className="formLogin" onSubmit={formik.handleSubmit}>
+          <h3>LOG IN</h3>
```

Comment:
```suggestion
       <h3>LOG IN</h3>
        <form className="formLogin" onSubmit={formik.handleSubmit}>
```
move h3 before form section